pieforproviders/pieforproviders
# [PIE-1658] definitions for remaining units columns

## 💅🏼 What issue does this fix?



Closes #1568 - adds definitions for the new columns

## 🍂 Before You Submit

* [X] Does your PR contain any required translations?

## 🧳 Steps to test

- Visit the dashboard with `nebraska@test.com` and `admin@test.com` to see the Nebraska dashboard
- Scroll all the way to the right
- The column header for the Total Authorized Period section should have a question mark indicating it's got definitions
- When you click the question mark, it should open the definitions
- The definitions should include a new section for Total Authorized Period with 3 definitions, for Authorized period, total hours remaining and total days remaining
- This should work in English and Spanish
- This should work in both accounts (admin & nebraska)
